datadog_sync/model/metric_percentiles.py

```diff
@@ -0,0 +1,58 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the 3-clause BSD style license (see LICENSE).
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+from typing import Optional, List, Dict, Tuple
+
+from datadog_sync.utils.base_resource import BaseResource, ResourceConfig
+from datadog_sync.utils.custom_client import CustomClient
+from datadog_sync.utils.resource_utils import SkipResource
+
+
+class MetricPercentiles(BaseResource):
+    resource_type = "metric_percentiles"
+    resource_config = ResourceConfig(
+        base_path="/metric/distribution/summary_aggr",
+        excluded_attributes=["key"],
+    )
+    # Additional MetricPercentiles specific attributes
+    metrics_summaries_get_path = "/metric/distribution/list_summaries"
+
+    async def get_resources(self, client: CustomClient) -> List[Dict]:
+        resp = await client.get(self.metrics_summaries_get_path)
```

Comment:
We should pass the `window` query param too to scope this query to just 14 days. It would look like this: `?window=86400`